src/lib/transform.js

```diff
@@ -62,6 +63,20 @@ function transformRelatedModel(field, prop) {
   return transformer;
 }
 
+function transformHTML(field) {
+  async function transformer(instance) {
+    const html = instance[field] || '';
+    const value = convert(html);
```

Comment:
I thin the Regex method would be faster, and allow for the implementation of https://ocio-jira.acf.hhs.gov/browse/TTAHUB-520

Reply:
After reading though the libraries examples, it does in fact retain formatting.  Good find.

Reply:
Please add table formatting:
`const value = convert(html, {
  tables: true
});`